src/templates/common/identifiers/crossref_conference.xml

```diff
@@ -4,17 +4,19 @@
                 {% if crossref_issue.issue.issue_title %}
                 <conference_name>{{ crossref_issue.issue.issue_title }}</conference_name>
                 {% else %}
-                <conference_name>{{ crossref_issue.journal.title }} - {{ crossref_issue.issue.year }}</conference_name>
+                <conference_name>{{ crossref_issue.journal.title }}</conference_name>
                 {% endif %}
             </event_metadata>
             <proceedings_metadata>
                 {% if crossref_issue.issue.issue_title %}
                 <proceedings_title>{{ crossref_issue.issue.issue_title }}</proceedings_title>
                 {% else %}
-                <proceedings_title>{{ crossref_issue.journal.title }} - {{ crossref_issue.issue.year }}</proceedings_title>
+                <proceedings_title>{{ crossref_issue.journal.title }}</proceedings_title>
```

Comment:
I think this can be a breaking change for a lot of folks. When a conference proceedings "journal" has articles for multiple years, they can't set a single journal title that works. I need the logic has to be altered on `inderifiers.logic` to cover the following cases:

 - When an article has a publication_title, that always takes precedece.
 - When a conference title is used and the issue has a year, preserve the existing functionality
 - when a conference title is used but no year is available in the issue, render just the title.

Reply:
I don't think so, because `issue.year` has never worked. There is no `journal.models.Issue.year` property or method.

This is the original commit that introduced it:
https://github.com/openlibhums/janeway/blob/eda3e9cf58b0142d218c21b789462f13949e707b/src/templates/common/identifiers/crossref_conference.xml#L32

This is a more recent commit that changed the line but preserved the bug:
https://github.com/openlibhums/janeway/blob/36e06831d72f5ba586e580faf567a7a3dc6e0705/src/templates/common/identifiers/crossref_conference.xml#L14

To chart a way forward, I think we should omit the year argument for now, and longer term create some of the metadata fields that would make it possible to structure this deposit in the way Crossref recommends:
 https://www.crossref.org/documentation/schema-library/markup-guide-record-types/conference-proceedings/